apps/src/p5lab/P5Lab.js

```diff
@@ -1572,12 +1594,13 @@ export default class P5Lab {
       // feedbackImage: feedbackImageCanvas.canvas.toDataURL("image/png")
       showingSharing: !level.disableSharing && level.freePlay,
       appStrings: {
-        reinfFeedbackMsg: msg.reinfFeedbackMsg(),
+        reinfFeedbackMsg,
```

Comment:
it's kind of linked in the PR description, but, more specifically, this setting/string is only used in this branch of FeedbackUtils.prototype.getFeedbackMessage:

https://github.com/code-dot-org/code-dot-org/blob/6be244bafd471fe390edba08077acdcb0db55c9e/apps/src/feedback.js#L891-L904